apps/src/music/analytics/AnalyticsReporter.ts

```diff
@@ -235,6 +237,10 @@ export default class AnalyticsReporter {
     this.trackUIEvent('Generate AI pattern start', {temperature});
   }
 
+  onChangeTuneScaleMode(scaleMode: ScaleMode) {
+    this.trackUIEvent('Change tune scale mode', {scaleMode});
```

Comment:
Maybe quick question for @dju90 on whether this should be tracked as its own event, or under the generic `'Key pressed'` event, with the event details in the `properties` payload?

Reply:
Makes sense to track it under the generic `button clicked` event with the event details in the `properties` payload

Reply:
Updated.